### DISCUSSION TITLE
Meeting 30.10.2023

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#52] [Weekly Meeting Agenda]
Wichtige Punkte : 
- Review User Story (Alle) #55 
- ERM Entities (Alle) #86 
- Technologies (Alle )  #35 
- Design (Marvin, Lara & Til) #10 
- Big Picture => Jere #13 
- Use Case Diagramme => Gerrit, Til #41 
- Statusbericht  => Jeremy Bents  #39 

Protokol zum Meeting:

Technology -> Kurze Aufteilung der Analysen für die Technologien :
Next.js => Gerrit, Lara
Prisma & MongoDb = Yannis
Nest.js => Marvin, Jere
TypeScript, Git/GitHub => Til Weber

ERM:
Extra Discussion erstellt für die entities
Jeder soll sich Gedanken machen und eventuelle Ideen Vorstelle

User Stories:

Ungefähre Beschreibung für die Umsetzung
Wenn möglich Mock Up
Aufwand & Risiken rausmachen
=> Jeder soll mal alles durchschauen
Big Picture => Jere
Use Case Diagramme => Gerrit, Til

Issues auf nächste Woche verschoben :)

Design Ideen / Mockup

- Dashboard als Homepage
- Mesaplan
- Adminseiten (Seite zum anmelden der Schüler etc.)
- Lehrerseiten (Notenvergabe etc.)
- Notification
- Noten
- Kalender
- News
- Krankmeldung
- Api-Seiten
- Sign in
- Einstellungen
- Klassenbuch

=> Mock Up nächste Woche
Jere würde gerne sich um HR kümmern

**Comment by laralngn** [comment]
Ich würde bei den User Stories noch generell die **Anforderungsanalyse** hinzufügen, da es jetzt ja Beispiele gibt und wir die langsam mal fertig machen sollten, damit wir anfangen können.
z.B. könnten wir die Beispiele einmal durchgehen und überlegen wie wir das machen wollen
ggf. noch Use Case Diagramme zu den Anforderungen hinzufügen

**Comment by laralngn** [comment]
Ziele zu den Design Ideen / Mockup wären vielleicht:
- Auf grundlegendes Design einigen (z.B. anhand von Vorschlägen aus Post)
- Groben Aufbau planen / skizzieren z.B. wie auf Bild oder als Text

=> Damit dann ein Plan besteht _**welche Seiten es alles geben soll und was grob auf den Seiten sein soll**_ / wie es dargestellt sein soll, damit wir es dann mit den Designs zusammen auf der Basis genauer ausgearbeitet können 

![image](https://github.com/SE-TINF22B2/G2-Schoolify/assets/115155205/51fcfc91-48ad-4a61-a1f7-43c716dfa155)

**Comment by Schmarvinius** [comment]
Sollen administrative Aktionen auch auf einem mobilen Gerät möglich sein?

**Comment by Kronprinz03** [comment]
Protokol zum Meeting:
Technology -> Kurze Aufteilung der Analysen für die Technologien :
Next.js => Gerrit, Lara 
Prisma & MongoDb = Yannis
Nest.js  => Marvin, Jere 
TypeScript, Git/GitHub => Til Weber 

ERM:
Extra Discussion erstellt für die entities 
Jeder soll sich Gedanken machen und eventuelle Ideen Vorstelle 

User Stories: 
- Ungefähre Beschreibung für die Umsetzung 
- Wenn möglich Mock Up
- Aufwand & Risiken rausmachen 
=> Jeder soll mal alles durchschauen 

Big Picture => Jere 
Use Case Diagramme => Gerrit, Til   

Issues auf nächste Woche verschoben :) 

Design Ideen / Mockup
- Dashboard als Homepage 
- Mesaplan  
- Adminseiten (Seite zum anmelden der Schüler etc.)
- Lehrerseiten (Notenvergabe etc.)
- Notification
- Noten 
- Kalender 
- News  
- Krankmeldung 
- Api-Seiten
- Sign in 
- Einstellungen
- Klassenbuch
=> Mock Up nächste Woche 

Jere würde gerne sich um HR
